.gitignore

```diff
@@ -55,3 +55,12 @@ public/sitemap*
 # Ignore local configuration
 .env.development
 .env.test
+
+# Ignore developer cruft
+**/.DS_Store
+.sass-cache
+
+# Ignore branding adjustments, user should track that separately
+# (ref app/assets/stylesheets/branding/README.md).
+app/assets/stylesheets/branding
+app/assets/images/branding
```

Comment:
I'm not sure that these should go in app. lib or vendor might be more appropriate, though I'm not entirely sure for this use case. See http://guides.rubyonrails.org/asset_pipeline.html#asset-organization.